scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/util/ChangeDBConfiguration.java

```diff
@@ -0,0 +1,686 @@
+/*
+ * ProActive Parallel Suite(TM):
+ * The Open Source library for parallel and distributed
+ * Workflows & Scheduling, Orchestration, Cloud Automation
+ * and Big Data Analysis on Enterprise Grids & Clouds.
+ *
+ * Copyright (c) 2007 - 2017 ActiveEon
+ * Contact: contact@activeeon.com
+ *
+ * This library is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU Affero General Public License
+ * as published by the Free Software Foundation: version 3 of
+ * the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * If needed, contact us to obtain a release under GPL Version 2 or 3
+ * or a different license than the AGPL.
+ */
+package org.ow2.proactive.scheduler.util;
+
+import java.io.*;
+import java.net.URISyntaxException;
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.apache.commons.cli.*;
+import org.apache.commons.io.IOUtils;
+import org.ow2.proactive.core.properties.PropertyDecrypter;
+import org.ow2.proactive.scheduler.core.properties.PASchedulerProperties;
+import org.ow2.proactive.utils.Tools;
+import org.ow2.proactive.web.WebProperties;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+
+
+public class ChangeDBConfiguration {
+
+    private static final String newline = System.lineSeparator();
+
+    public static final String HELP_OPTION = "h";
+
+    public static final String HELP_OPTION_NAME = "help";
+
+    public static final String DB_VENDOR_OPTION = "v";
+
+    public static final String DB_VENDOR_OPTION_NAME = "database-vendor";
+
+    public static final String COMPONENT_OPTION = "c";
+
+    public static final String COMPONENT_OPTION_NAME = "target";
+
+    public static final String URL_OPTION = "r";
+
+    public static final String URL_OPTION_NAME = "url";
+
+    public static final String USERNAME_OPTION = "u";
+
+    public static final String USERNAME_OPTION_NAME = "username";
+
+    public static final String PASSWORD_OPTION = "p";
```

Comment:
-P for password
and
-p for port
isn't better than -p and -t  ?

Reply:
I forgot that it was possible to use uppercase latter. I would use the opposite though, as I intend to do the same for hostname

Reply:
Using the same letter for both options is not good for me (even with capital letters).
We can use more than one letter, for example `-pwd` for password

Reply:
@nebilbenmabrouk say that to the linux developpers, many linux commands have parameters in both lowercase and uppercase.

I think it's perfectly fine to use uppercase letters

Reply:
I agee with Fabien I just looked at scp command as example, it is the case